quasar/chain.json

```diff
@@ -19,128 +20,71 @@
       }
     ]
   },
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ibc/0471F1C4E7AFD3F07702BEF6DC365268D64570F7C1FDC98EA6098DD6DE59817B",
```

Comment:
I don't know what these three tokens are, but could we register them as ics20 assets and add register their corresponding ibc channels?

Reply:
Hey @JeremyParish69 
Chain is only accepting those denoms for gas. Native token will be launch after a few weeks.
How do we register them as ics20 assets?

We will be also registering the corresponding IBC channels soon when we visit

Reply:
@pabloqsr we ended up discussing adding the ICS tokens off-line; you can see them in the assets file. No more is needed here at this time